plugins/inputs/sqlserver/README.md

```diff
@@ -56,6 +61,27 @@ GO
   # azuredb = false
```

Comment:
add parameter outdated/deprecated info
```md
  ## The "azuredb" parameter is deprecated, use the new "database_type" parameter.
  ## If you are using AzureDB, setting this to true will gather resource utilization metrics
  # azuredb = false
```

Reply:
Done

Reply:
I don't any comment about the deprecation of "azuredb" parameter (at least on the parameter itself)